### DISCUSSION TITLE
How to show shadow?

### DISCUSSION THREAD

**Discussion by Yurii-huang** [Textualize/textual#4034] [Q&A] [answered]
```css
widget {
     background: #aaaaaa;
     box-shadow: ?????
}
```

not supportion?

**Comment by davep** [comment, marked as answer]
You can find [all of the styles that are part of Textual's CSS listed in the documentation](https://textual.textualize.io/styles/). There isn't a style called `box-shadow` and Textual doesn't have shadows as a feature.
